ote/test/clj/ote/integration/export/geojson_test.clj

```diff
@@ -36,7 +36,8 @@
   ;; Make it public so that GeoJSON export can return it
   (sql-execute! "UPDATE \"transport-service\" SET published = to_timestamp(0) where id = " id)
   (let [geojson-url (str "export/geojson/" transport-operator-id "/" id)
-        json-response (http-get "admin" geojson-url)]
+        json-response (http-get (fetch-id-for-username (:db ote.test/*ote*) "admin")
```

Comment:
Just wondering if (fetch-id-for-username) could be done once with fixture :once

Reply:
Good point, will do a PR about that.